### PR TITLE
scripts: Add script to review the highlighted IDs

### PR DESCRIPTION
**Pull request by manuq**
https://phabricator.endlessm.com/T33090

### CODE REVIEW

**Review by erikos — COMMENTED** [review]
Looks good to me high level. Very nice and useful tool!

File: scripts/review_highlighted.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+content_ids = None
+with open(HIGHLIGHTED_CONTENT_PATH, "r") as input_file:
+    data = json.load(input_file)
+    content_ids = list(chain.from_iterable(data.values()))
+
+# Output error for each content ID not found in the database:
+query = "SELECT title FROM content_contentnode WHERE id = ?"
+for content_id in content_ids:
+    cursor.execute(query, (content_id,))
+    exists = len(cursor.fetchall()) > 0
```
Comment (dylanmccall) [inline scripts/review_highlighted.py:30]: Complete non-issue, but this would be a tiny bit faster if it used `cursor.fetchone()` ;)

**Review by dylanmccall — APPROVED** [review]
This works for me! And I appreciate that it's simple and straightforward to use. Nice stuff :)

File: scripts/review_highlighted.py (hunk 1 above)
Comment (manuq) [reply scripts/review_highlighted.py:30]: Oh good to know! Thanks, I will change it tomorrow.

**Pull request merged**